tensorboard/components_polymer3/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -91,12 +91,25 @@ export function DataLoaderBehavior<Item, Data>(
 
     dataLoading = false;
 
+    // The standard Node.isConnected doesn't seem to be set reliably, so we
+    // wire up our own property manually.
+    _isConnected = false;
+
+    connectedCallback() {
+      this._isConnected = true;
```

Comment:
Did you manually test this? You may need to call `super`.

Reply:
Thanks for the catch - I did test it manually and it definitely worked, but who knows...